src/helpers/user/formValidator.js

```diff
@@ -0,0 +1,3 @@
+export const regularExpression = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+
+export const emailError = 'email must be a valid email';
```

Comment:
do we really need this message here, I can see it is not needed

Reply:
No need, I have already removed

Reply:
Good